### DISCUSSION TITLE
Help setting up REM first timer and adding the 1st valve.

### DISCUSSION THREAD

**Discussion by bwinstonbillville** [tagyoureit/nodejs-poolController#985] [REM]
First off _thanks for all of the hardwork that has gone into this project!

I have my first version of the hardware build done.  Pi loaded and the poolController up and relayEquipment Manager up- YAY

I have the Sequent MEGA-BAS and the Sequent 8 relay HAT setup in REM.  I'm trying to get my first valve setup - the intake valve on relay 1.  Can you walk me through what's needed?  Should I be able to click the first relay button and get the valve to switch positions?

![image](https://github.com/tagyoureit/nodejs-poolController/assets/172343770/66cf2458-8233-442e-8621-8db9b3e6b7a4)

I tried for a very long time before I posted a question.  I did some searches and I just can't find the info..

**Comment by rstrouse** [comment]
Is this a pool/spa combo?  If it is you should have an intake and return valve defined in REM.   This will control the switchover from pool and spa as well as the spillway feature.  Once you have that then you assign the relay in dashPanel.  Here is a wiki on valves.

https://github.com/tagyoureit/nodejs-poolController/wiki/Nixie-Controller-Setup#valves

**Reply by bwinstonbillville** [reply]
Yes Pool/Spa combo

I don't know how to get into dashPanel - feeling kind of dumb here.

poolController is ip:5150
REM is ip:8180
how do you pull up dashPanel?

Thank you for your patience.  I will hopefully be able to pick up on this stuff quicker

**Reply by rstrouse** [reply]
dashPanel is the user interface under :5150.   Click on the gears menu on the upper right corner then click on valves.

**Reply by bwinstonbillville** [reply]
Thank you.  I have done that - didn't realize the dashPanel was the name of the gui for poolController.  Makes sense now.

I had configured the relays in REM but did not see the Intake and Return names can't be changed.

I there a wiki for setting up the temperature probes?  I have them setup in REM, but I'm not seeing the temp or where to select the REM input from the MEGA-BAS.

Thanks again for your help and your patience.

**Reply by rstrouse** [reply]
Here you go.  I assume you have these set up on your BAS.

https://github.com/rstrouse/relayEquipmentManager/wiki/Sequent-Microsystems-Boards#sequent-building-automation-hat-mega-bas
